github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; need-more-info (optional) -&gt; understood (optional) -&gt; in -progress -&gt; Closed. *
+
+****Enhancement Happy Path****
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; design -&gt; in -progress -&gt; review -&gt; Closed. *
+
+> ****Questions, Defects and Doc issues****
+>
+> ***'Optional' Phases will apply to issues that are immediately understood allowing them to bypass many of the phases, moving the bug right to 'in-progress'.***
+
+****Enhancement Issues****
+
+> ***Where at 'review' the feature is reviewed for release and obtains any special sign-off needed from feature teams (if applicable). If not done (completed or has issues), go back to investigate (if needed) or move to design or in-progress for additional work. Repeat as needed. Once the feature can pass the review process (Whatever that may be for the given feature) it moves to closed***
+
+Issue Lifecycle
+===============
+
+Issues are reported with no labels, and no assignees. Issues then typically follow this cycle:
+
+1.  When an issue is reported it is sent to an appropriate area using the area label
+
+2.  If the type of issue is obvious, a type label is set
+
+3.  If the issue is not obvious, an investigating status label is set, and assignee is set
+
+4.  The individual assigned works on the issue, updating the issue, status and type labels
+
+If the assignee has found the issue does not belong to their area, they initiate a transfer by
+
+1.  Removing the area label
+
+2.  Removing themselves from assignee
+
+3.  Transferring issue to new area label
+
+Assigning issues
+================
+
+> Issues should only be assigned to individuals if they are actively being worked on by that individual. Otherwise, issues remain unassigned. This way it is obvious who is responsible for the work. It is valid for no one to be responsible.
+
+Work Flow Process Details
+=========================
+
+Who are all those involved with the project?
+--------------------------------------------
+
+Inputs will come from both **Internal** and **External** service-fabric repo contributors and community members. Here are some of the roles we'll see.
+
+| **Area Leads**:   | Responsible for setting direction in specific areas of the project and triaging issues within their respective areas.|
+|-|-
+| **Committers**:   | Dedicated to working on the project, may have write access.|
+| **Contributors**: | Community members that make any contribution to the project, whether it’s submitting code, fixing or reporting bugs, writing docs, etc.|
+| **Users:**        | General users who have a need for the project.|
+
+> **As a measurement** we want to try and touch all new items within 48 hours. This is not exposed to the customer but is a metric for us to determine how quickly we are touching new issues. We need to ensure that feature area owners are actively triaging their issues and keeping them up to date.
```

Comment:
This document is on public github, so yes it is exposed to the customer.  Also, while there will be "customers" using the product, there are also "contributors" who are helping build the product.  We should be considerate to not call everyone who is filing issues a "customer" because they may very well be a contributor who wants to be part of the development community and work **on** service fabric, as opposed to just **using** service fabric.